### PR TITLE
Ydata synthetic

### PR DESCRIPTION
**Pull request by rxm7706**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Comment by rxm7706** [comment]
@conda-forge/staged-recipes  ready for review

File: recipes/ydata-synthetic/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - matplotlib-base >=3.5.0
+    - tensorflow-cpu >=2.8.0
+    - easydict >=1.9
+    - pmlb >=1.0
+    - tqdm
+    - typeguard >=2.13.0
+    - pytest >=6.2.0
+    - requests >=2.24.0
+
+test:
```
Comment (xhochy) [inline recipes/ydata-synthetic/meta.yaml:35]:
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/ydata-synthetic/meta.yaml (hunk 1 above)
Comment (rxm7706) [reply recipes/ydata-synthetic/meta.yaml:35]: Pip Check will be very restrictive and fail, as the requirements.txt is too locked down
requests>=2.24.0, <2.29
pandas==1.4.*
numpy==1.23.*
scikit-learn==1.1.*
matplotlib==3.5.*
tensorflow==2.9.0
easydict==1.9
pmlb==1.0.*
tqdm<5.0
typeguard==2.13.*
pytest==6.2.*

Comment (xhochy) [reply recipes/ydata-synthetic/meta.yaml:35]: Please then raise an issue about this upstream and mention it here.

**Pull request closed without merging**